compiler/rustc_error_messages/locales/en-US/parser.ftl

```diff
@@ -46,6 +46,10 @@ parser_invalid_comparison_operator = invalid comparison operator `{$invalid}`
     .use_instead = `{$invalid}` is not a valid comparison operator, use `{$correct}`
     .spaceship_operator_invalid = `<=>` is not a valid comparison operator, use `std::cmp::Ordering`
 
+parser_invalid_shift_operator = invalid shift operator `{$invalid}`
+    .arithmetic_left_shift_operator_invalid = `<<<` is not a valid left shift operator, consider shifting normally and fixing the sign as needed.
+    .arithmetic_right_shift_operator_invalid = `>>>` is not a valid right shift operator, consider casting to a signed integer and right shifting normally.
```

Comment:
We don't end sentences with period (nor we capitalize the first letter in sentences)

```suggestion
    .arithmetic_left_shift_operator_invalid = `<<<` is not a valid left shift operator, consider shifting normally and fixing the sign as needed
    .arithmetic_right_shift_operator_invalid = `>>>` is not a valid right shift operator, consider casting to an unsigned integer and right shifting normally
```

I would prefer if the `consider...` text was instead a `note`, separate from the main label

Reply:
I agree with both of these comments.